extras/data/JLABE06/metadata.yaml

```diff
@@ -0,0 +1,5 @@
+JLABE06:
+  description: Measurements with Polarised beams
+  nnpdf_id: JLABE06
+  y_label: G1F1
+  y_label_tex: $g1/f1$
```

Comment:
this still does not make sense: it is either g1 or F1

Reply:
This is precisely why this metadata is confusing to me... 
This dataset is really $g1/f1$, now if you prefer to label it as $g1,f1$ fine.

Reply:
I would be prefer either `y_label_tex: $g_1$` or `y_label_tex: $F_1$` - no additional character what so ever, because it is really one or the other

Reply:
> As right now `y_label` is unused.

no - of course pineappl will use it

Reply:
I think this makes sense, but once we apply the k-factors $1/(2xF_1)$ the final FK tables are really $g_1/F_1$. So we need to overwrite the metadata again at that stage.

Reply:
of course, if that is so! we need to state what it actually is :upside_down_face:

Reply:
> As they are now they are simply ignored in any case.

maybe in yadism, but all this is going outside and there people want to know; people will ask pinefarm for `E155_NC_9GEV_EP_F1` and then the thing should faithfully report what it is

> description: $g_1$ component of $g_1/F_1$ ratio neutron from E155

how about "polarized structure function $g_1$ in E155 with neutron target"? (or similar)

Reply:
> maybe in yadism, but all this is going outside and there people want to know; people will ask pinefarm for `E155_NC_9GEV_EP_F1` and then the thing should faithfully report what it is

I meant they are ignored by  `machinery` i.e. the information are not propagated because they are under a wrong key.

> "polarized structure function $g_1$ in E155 with neutron target"

okay fine. Not sure about the role of `nnpdf_id` ...

Reply:
> I meant they are ignored by `machinery` i.e. the information are not propagated because they are under a wrong key.

mh? `machinery` needs to copy the correct information to the generated `metadata.txt` - that will be part of the pinecard, so yes it does matter

> Not sure about the role of `nnpdf_id` ...

I think that is legacy and with the merge of new CD now completely ignored everywhere, I believe - we can ask JCM for confirmation if you wish

Reply:
Okay if `nnpdf_id` and `fktable_id` are not used anymore I'd not add them. Is it correct @Radonirinaunimi, @scarlehoff

Reply:
Yes, those two are not longer used in the new CD and therefore deprecated so should be removed.

Reply:
These were also not used in the old one either. It was mostly (only) to keep track of for which dataset the runcards were generated but afaik it was never actually used for anything other than bookkeping